.vscode/settings.json

```diff
@@ -1,3 +1,15 @@
 // Place your settings in this file to overwrite default and user settings.
```

Comment:
I really think this does not belong in the repo. Even though most dev may happen it vscode, that does not mean everybody will want to have the same settings. So if I want to change a setting in settings.json, I can't check it in unless everybody else wants to have the same change. Unless there is a settings.user.json which can be ignored, while the main file contains common settings?